plugins/modules/azure_rm_openshiftmanagedcluster_info.py

```diff
@@ -0,0 +1,382 @@
+        elif (self.resource_group is not None):
+            self.results['clusters'] = self.list()
+        else:
+            self.results['clusters'] = self.listall()
+        return self.results
+
+    def get(self):
+        response = None
+        results = {}
+        # prepare url
+        self.url = ('/subscriptions' +
+                    '/{{ subscription_id }}' +
+                    '/resourceGroups' +
+                    '/{{ resource_group }}' +
+                    '/providers' +
+                    '/Microsoft.RedHatOpenShift' +
+                    '/openShiftClusters' +
+                    '/{{ open_shift_managed_cluster_name }}')
+        self.url = self.url.replace('{{ subscription_id }}', self.subscription_id)
+        self.url = self.url.replace('{{ resource_group }}', self.resource_group)
+        self.url = self.url.replace('{{ open_shift_managed_cluster_name }}', self.name)
+
+        try:
+            response = self.mgmt_client.query(self.url,
+                                              'GET',
+                                              self.query_parameters,
+                                              self.header_parameters,
+                                              None,
+                                              self.status_code,
+                                              600,
+                                              30)
+            results = json.loads(response.text)
+            # self.log('Response : {0}'.format(response))
+        except CloudError as e:
+            self.log('Could not get info for @(Model.ModuleOperationNameUpper).')
+
+        return self.format_item(results)
+
+    def list(self):
+        response = None
+        results = {}
+        # prepare url
+        self.url = ('/subscriptions' +
+                    '/{{ subscription_id }}' +
+                    '/resourceGroups' +
+                    '/{{ resource_group }}' +
+                    '/providers' +
+                    '/Microsoft.RedHatOpenShift' +
+                    '/openShiftClusters')
+        self.url = self.url.replace('{{ subscription_id }}', self.subscription_id)
+        self.url = self.url.replace('{{ resource_group }}', self.resource_group)
+
+        try:
+            response = self.mgmt_client.query(self.url,
+                                              'GET',
+                                              self.query_parameters,
+                                              self.header_parameters,
+                                              None,
+                                              self.status_code,
+                                              600,
+                                              30)
+            results = json.loads(response.text)
+            # self.log('Response : {0}'.format(response))
+        except CloudError as e:
+            self.log('Could not get info for @(Model.ModuleOperationNameUpper).')
```

Comment:
If the exception does not catch a value at this location, an empty list should be returned.
```suggestion
            self.log('Could not get info for @(Model.ModuleOperationNameUpper).')
            retrun []
```